integrationTests/chainSimulator/relayedTx/relayedTx_test.go

```diff
@@ -25,12 +28,11 @@ import (
 	"github.com/multiversx/mx-chain-go/process"
 	"github.com/multiversx/mx-chain-go/sharding"
 	"github.com/multiversx/mx-chain-go/vm"
-	logger "github.com/multiversx/mx-chain-logger-go"
-	"github.com/stretchr/testify/require"
 )
 
 const (
 	defaultPathToInitialConfig              = "../../../cmd/node/config/"
+	defaultPathToSovereignConfig            = "../../../cmd/sovereignnode/config/"
```

Comment:
This sould've been moved to the `integrationTests/chainSimulator/relayedTx/sovereignRelayedTx_test.go`